CorePluginFramework/src/au/gov/asd/tac/constellation/pluginframework/parameters/types/SingleChoiceParameterType.java

```diff
@@ -170,6 +170,7 @@ public static ParameterValue getChoiceData(final PluginParameter<SingleChoicePar
 
     /**
      * Set the current selected choice from a string.
+     * This will fire a {@link ParameterChange} event if successful.
```

Comment:
If you are going to add this to `SingleChoiceParameterType` can you add it to the other *ParameterType classes for consistency? Otherwise removing it is also OK.